src/course-outline/configure-modal/messages.js

```diff
@@ -0,0 +1,50 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  title: {
+    id: 'course-authoring.course-outline.configure-modal.title',
+    defaultMessage: '{title} Settings',
+  },
+  basicTabTitle: {
+    id: 'course-authoring.course-outline.configure-modal.basic-tab.title',
+    defaultMessage: 'Basic',
+  },
+  releaseDateAndTime: {
+    id: 'course-authoring.course-outline.configure-modal.basic-tab.release-date-and-time',
+    defaultMessage: 'Release Date and Time',
+  },
+  releaseDate: {
+    id: 'course-authoring.course-outline.configure-modal.basic-tab.release-date',
+    defaultMessage: 'Release Date:',
+  },
+  releaseTimeUTC: {
+    id: 'course-authoring.course-outline.configure-modal.basic-tab.release-time-UTC',
+    defaultMessage: 'Release Time in UTC:',
```

Comment:
Remove `in UTC` and use `showUTC` in the component instead

Reply:
Please correct me if I am wrong. But wouldn't it be better to keep the same string so we can leverage the existing translations?

Reply:
Ack, makes sense. Let's keep the string :)